pkg/gui/controllers/bisect_controller.go

```diff
@@ -91,25 +99,39 @@ func (self *BisectController) openMidBisectMenu(info *git_commands.BisectInfo, c
 			Key: 'g',
 		},
 		{
-			Label: fmt.Sprintf(self.c.Tr.Bisect.Skip, commit.ShortSha()),
+			Label: fmt.Sprintf(self.c.Tr.Bisect.SkipCurrent, shortShaToMark),
 			OnPress: func() error {
 				self.c.LogAction(self.c.Tr.Actions.BisectSkip)
-				if err := self.c.Git().Bisect.Skip(commit.Sha); err != nil {
+				if err := self.c.Git().Bisect.Skip(shaToMark); err != nil {
 					return self.c.Error(err)
 				}
 
 				return self.afterMark(selectCurrentAfter, waitToReselect)
 			},
 			Key: 's',
 		},
-		{
-			Label: self.c.Tr.Bisect.ResetOption,
+	}
+	if info.GetCurrentSha() != "" && info.GetCurrentSha() != commit.Sha {
+		menuItems = append(menuItems, lo.ToPtr(types.MenuItem{
```

Comment:
When we're bisecting (meaning there is a 'current' commit), what are your thoughts on showing both skip options unconditionally, for the sake of muscle memory?

Reply:
I'd vote against it. In the vast majority of cases, the selected commit and the current commit will be the same, so it would be confusing to always see two menu entries for the same sha that only differ in their keyboard shortcut. Also (in my personal experience, at least) it's much more common to skip the current commit rather than some arbitrary different one, so most people will have `s` in their muscle memory and not `S`.

Reply:
Fair point: you've convinced me!